google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -544,19 +570,9 @@ private EnhancedBigtableStub(
 
     UnaryCallable<RequestT, ResponseT> traced =
         new TracedUnaryCallable<>(
-            inner,
-            clientContext.getTracerFactory(),
-            SpanName.of(TRACING_OUTER_CLIENT_NAME, methodName));
-
-    UnaryCallable<RequestT, ResponseT> measured =
-        new MeasuredUnaryCallable<>(
-            traced,
-            TRACING_OUTER_CLIENT_NAME + "." + methodName,
-            tagger,
-            statsRecorder,
-            clientContext.getClock());
+            inner, clientContext.getTracerFactory(), SpanName.of(CLIENT_NAME, methodName));
 
-    return measured.withDefaultCallContext(clientContext.getDefaultCallContext());
+    return traced.withDefaultCallContext(clientContext.getDefaultCallContext());
   }
   // </editor-fold>
```

Comment:
super minor, but while you're here can you delete these random comments?

Reply:
I find those comments useful, they allow me to enable code folding in intellij

Reply:
ohh, a feature I didn't know existed :) Sounds good then!